mkdocs.yml

```
@@ -157,6 +157,82 @@ nav:
+              - Circuit Type: "models/circuits/circuittype.md"
+              - Circuit Provider: "models/circuits/provider.md"
+              - Circuit Provider Network: "models/circuits/providernetwork.md"
+          - DCIM:
+              - Cable: "models/dcim/cable.md"
+              - Console Port: "models/dcim/consoleport.md"
+              - Console Port Template: "models/dcim/consoleporttemplate.md"
+              - Console Server Port: "models/dcim/consoleserverport.md"
+              - Console Server Port Template: "models/dcim/consoleserverporttemplate.md"
+              - Device: "models/dcim/device.md"
+              - Device Bay: "models/dcim/devicebay.md"
+              - Device Bay Template: "models/dcim/devicebaytemplate.md"
+              - Device Redundancy Group: "models/dcim/deviceredundancygroup.md"
+              - Device Role: "models/dcim/devicerole.md"
+              - Device Type: "models/dcim/devicetype.md"
+              - Front Port: "models/dcim/frontport.md"
+              - Front Port Template: "models/dcim/frontporttemplate.md"
+              - Interface: "models/dcim/interface.md"
+              - Interface Template: "models/dcim/interfacetemplate.md"
+              - Inventory Item: "models/dcim/inventoryitem.md"
+              - Location: "models/dcim/location.md"
+              - Location Type: "models/dcim/locationtype.md"
+              - Manufacturer: "models/dcim/manufacturer.md"
+              - Platform: "models/dcim/platform.md"
+              - Power Feed: "models/dcim/powerfeed.md"
+              - Power Outlet: "models/dcim/poweroutlet.md"
+              - Power Outlet Template: "models/dcim/poweroutlettemplate.md"
+              - Power Panel : "models/dcim/powerpanel.md"
+              - Power Port: "models/dcim/powerport.md"
+              - Power Port Template: "models/dcim/powerporttemplate.md"
+              - Rack: "models/dcim/rack.md"
+              - Rack Group: "models/dcim/rackgroup.md"
+              - Rack Reservation: "models/dcim/rackreservation.md"
+              - Rack Role: "models/dcim/rackrole.md"
+              - Rear Port: "models/dcim/rearport.md"
+              - Rear Port Template: "models/dcim/rearporttemplate.md"
+              - Region: "models/dcim/region.md"
+              - Site: "models/dcim/site.md"
+              - Virtual Chassis: "models/dcim/virtualchassis.md"
+          - Extras:
+              - Config Context: "models/extras/configcontext.md"
+              - Config Context Schema: "models/extras/configcontextschema.md"
+              - GraphQL Query: "models/extras/graphqlquery.md"
+              - Image Attachment: "models/extras/imageattachment.md"
+              - Job: "models/extras/job.md"
+              - Job Log Entry: "models/extras/joblogentry.md"
+              - Job Result: "models/extras/jobresult.md"
+              - Secret: "models/extras/secret.md"
+              - Secrets group: "models/extras/secretsgroup.md"
```

```suggestion
              - Role: "models/extras/role.md"
              - Secret: "models/extras/secret.md"
              - Secrets group: "models/extras/secretsgroup.md"
```

Looks like we're also missing `Status`, `Tag` and `Webhook` - but that should really get fixed in develop rather than here.